report_generator/outputs/full_report.md

```diff
@@ -0,0 +1,265 @@
+# Code report
+
+**Repository link:** https://repository.byont/
+
+**Commit number:** j230a-al4uf-zmh2
+
+**Relative path to contract in the repository being verified:** https://contract-path-url.byont/
```

Comment:
Doesn't seem very interesting to me; It can also be a set of contracts right?

Reply:
Most verification tools are considering only a single contract in a single run. How external calls are handled depends per tool.

Reply:
Though there's a difference between the verification tools we employ to gather the data, and the outputs we might want to achieve. 

Have a look at openzeppelins audits:
https://blog.openzeppelin.com/origin-dollar-audit-2/
They list a set of contracts which they audited